rules/sql-indexing-where/rule.md

```diff
@@ -0,0 +1,161 @@
+---
+seoDescription: How to index SQL WHERE clauses.
+type: rule
+archivedreason:
+title: Do You Know How to Index SQL WHERE Clauses?
```

Comment:
Hi @brydeno 

1. Can you fix the Capitalization in this title to be standard?

Reply:
Done